src/assets/simp-adapter/build/simp-adapter.spec

```diff
@@ -214,7 +211,7 @@ puppet config set stringify_facts false || :
 (
   cd %{puppet_confdir}
 
-  simp_overrides='hiera.yaml auth.conf'
+  simp_overrides='hiera.yaml'
```

Comment:
Was this the only mechanism to back up an existing auth.conf file?  I'm concerned that we will silently wipe out an existing file with the use of the Puppet auth.conf resource, upon upgrade.  If we can back it up, that would be great. If that is not feasible, this should be in the CHANGELOG.